src/app/components/Homepage.js

```diff
@@ -1,15 +1,19 @@
 import React from 'react';
 
 import Grid from '@material-ui/core/Grid';
+import { Provider } from 'react-redux';
 
-import { CompanyForm } from '../containers/CompanyForm';
+import CompanyForm from '../containers/CompanyForm';
+import store from '../store/store';
 
 const Homepage = () => {
   return (
     <Grid container justify="center">
       <Grid item xs={10} sm={8} md={6}>
         <h1>Bridge Company Certification</h1>
-        <CompanyForm />
+        <Provider store={store}>
```

Comment:
Hey why do we have 2 Providers? 🤔

Reply:
These two Providers is something I wanted to ask you about in class. I could not make it work without adding this second Provider - the only test in the app is failing without it. Even though the app seemed to be running with no troubles. Does this error sound familiar at all to you?

`Error: Uncaught [Invariant Violation: Could not find "store" in the context of "Connect(Form(CompanyForm))". Either wrap the root component in a <Provider>, or pass a custom React context provider to <Provider> and the corresponding React context consumer to Connect(Form(CompanyForm)) in connect options.]`